Test/dafny0/SubsetTypes.dfy

```diff
@@ -228,21 +228,21 @@ module AssignmentsFromNewAllocation {
       b := a;  // error: ditto
     }
   }
-
+*/
   method M(N: nat, p: Person) {
     var a: array<Person?>;
     var b: array<Person>;
     if * {
-      a := b;  // error
+      // a := b;  // error
```

Comment:
Yes, good! The commented-out tests in this method should indeed be caught as type errors. Since they are (and are tested as such in `SubsetTypesERR.dfy`), I think the tests are better just deleted here (rather than being commented out--although being commented out helped do this review).

Reply:
Sorted out the two test files

Reply:
revised these tests